assets/js/romo/sortable.js

```diff
@@ -12,80 +12,82 @@ var RomoSortable = function(element) {
-  draggableElems.prop('draggable', 'true');
+  draggableElems.forEach(Romo.proxy(function(draggableElem) {
+    draggableElem.draggable = true;
+  }, this));
 
-  draggableElems.on('dragstart',  $.proxy(this.onDragStart,          this));
-  draggableElems.on('dragenter',  $.proxy(this.onDragEnter,          this));
-  draggableElems.on('dragover',   $.proxy(this.onDragOver,           this));
-  draggableElems.on('dragend',    $.proxy(this.onDragEnd,            this));
-  draggableElems.on('drop',       $.proxy(this.onDragDrop,           this));
-  draggableElems.on('mousedown',  $.proxy(this.onDraggableMouseDown, this));
+  Romo.on(draggableElems, 'dragstart',  Romo.proxy(this.onDragStart,          this));
+  Romo.on(draggableElems, 'dragenter',  Romo.proxy(this.onDragEnter,          this));
+  Romo.on(draggableElems, 'dragover',   Romo.proxy(this.onDragOver,           this));
+  Romo.on(draggableElems, 'dragend',    Romo.proxy(this.onDragEnd,            this));
+  Romo.on(draggableElems, 'drop',       Romo.proxy(this.onDragDrop,           this));
+  Romo.on(draggableElems, 'mousedown',  Romo.proxy(this.onDraggableMouseDown, this));
 
-  var handleElems = draggableElems.find(this.handleSelector);
-  handleElems.on('mousedown', $.proxy(this.onHandleMouseDown, this));
+  var handleElems = Romo.find(draggableElems, this.handleSelector);
+  Romo.on(handleElems, 'mousedown', Romo.proxy(this.onHandleMouseDown, this));
 
-  this.draggableElems = this.draggableElems.add(draggableElems);
+  this.draggableElems = this.draggableElems.concat(draggableElems);
   this._resetGrabClasses();
 }
 
 RomoSortable.prototype.doInitPlaceholder = function() {
   var tag;
   try {
-    tag = this.draggableElems.get(0).tagName;
+    tag = this.draggableElems[0].tagName;
   } catch(e) {
     tag = /^ul|ol$/i.test(this.elem.tagName) ? 'li' : 'div';
   }
-  this.placeholderElem = $('<' + tag + '/>');
-  this.placeholderElem.addClass(this.placeholderClass);
+  this.placeholderElem = Romo.elems('<' + tag + '/>')[0];
+  Romo.addClass(this.placeholderElem, this.placeholderClass);
 
-  this.placeholderElem.on('dragover', $.proxy(this.onDragOver, this));
-  this.placeholderElem.on('drop',     $.proxy(this.onDragDrop, this));
+  Romo.on(this.placeholderElem, 'dragover', Romo.proxy(this.onDragOver, this));
+  Romo.on(this.placeholderElem, 'drop',     Romo.proxy(this.onDragDrop, this));
 }
 
 RomoSortable.prototype.onDragStart = function(e) {
   if(!this.draggableSelected){ return false; }
 
   e.stopPropagation();
-  e.originalEvent.dataTransfer.effectAllowed = 'move';
+  e.dataTransfer.effectAllowed = 'move';
 
   // IE fix
   try {
     // FF fix, it won't drag without some data being set
-    e.originalEvent.dataTransfer.setData('text/plain', null);
+    e.dataTransfer.setData('text/plain', null);
   } catch(e) {}
 
-  this.draggedElem = $(e.target);
-  this.draggedElem.addClass(this.draggingClass);
+  this.draggedElem = Romo.f(e.target)[0];
```

Comment:
@jcredding don't think you need to do this here, instead just `this.draggedElem = e.target;`